### PR TITLE
Added index webpage

### PR DESCRIPTION
**Pull request by nicholasbay**

### CODE REVIEW

**Comment by barbprawn** [comment]
COMMENt

File: index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```
Comment (lcwlouis) [inline index.html:3]: Ahh

File: index.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
```
Comment (lcwlouis) [inline index.html:9]: Well Yes this works well finally found it

File: index.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>My First Github Repository</title>
+  </head>
+  <body>
+    <nav class="navbar navbar-light bg-dark justify-content-center align-items-center">
```
Comment (lcwlouis) [inline index.html:17]: WOW THIS IS SO COOL

File: index.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    </nav>
+    <div class="container my-5">
+      <h1 class="text-center">Welcome!</h1>
+      <p class="text-center">
+        This is my first Github repository. I am excited to share my projects
+        with you.
+      </p>
+    </div>
+  </body>
+</html>
```
Comment (lcwlouis) [inline index.html:30]: Goodnight

**Review by lcwlouis — COMMENTED** [review]
TYVM

**Pull request merged**